src/Lab.Quiz.DAL/Repositories/Repository.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Microsoft.EntityFrameworkCore;
+using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Lab.Quiz.DAL.Interfaces;
+using Lab.Quiz.DAL.Entities;
+using System.Collections.Generic;
+
+namespace Lab.Quiz.DAL.Repositories
+{
+    public class Repository<TEntity> : IRepository<TEntity> where TEntity : class 
+    {
+        private DbContext _dbContext;
+        DbSet<TEntity> Entities { get; set; }
+        public Repository(DbContext dbContext)
```

Comment:
Again, there is no need to have a DbContext dependency in this class. As far as it is going to use only DbSet's things, just inject DbSet<TEntity> directly.

Reply:
Like this?
![image](https://user-images.githubusercontent.com/61378851/112634848-25eed580-8e4c-11eb-8e3e-bf0b64ed6014.png)